### ISSUE TITLE
Shift-M does not work for merging cells

### ISSUE THREAD

**Issue by gully** [jupyter/jupyter#64]
Shift-M does not work for merging cells.  It used to work for me all the time, has something changed?  I'm in command mode (esc).  Sorry if this is a duplicate!

**Comment by Carreau** [comment]
use Shift-J/K of Shift-Up/Down to select many cells, and then Shift-M to merge the selected one.

**Comment by andaag** [comment]
Merge next might be a sane default for Shift+M if no other cells are detected

**Comment by gully** [comment]
Thanks, both!  This worked for me, it looks like the behavior changed from IPython to Jupyter.  I think the current default (no-op if only one cell is selected) is fine for me now.  Though it is true that my typical use case is to merge the cell immediately below.  Anyways, thanks again!

**Comment by takluyver** [comment]
I think right now a lot of people are used to the way it worked before, and would be happy if it merged the cell below when only one cell is selected. But in the longer term, I think it's an easier mental model for users to grasp without a special case for a single selected cell.

We had developed a few bits of rather awkward UX like this when there could only be one cell selected, and I don't want them to linger now that we've removed that limitation.
